Formula/wasm-pack.rb

```diff
@@ -24,6 +29,7 @@ def install
 
     system "#{Formula["rustup-init"].bin}/rustup-init", "-y", "--no-modify-path"
     ENV.prepend_path "PATH", HOMEBREW_CACHE/"cargo_cache/bin"
+    ENV.prepend_path "PATH", Formula["binaryen"].opt_bin
```

Comment:
Wouldn't this be standard when you add a dependency?

Reply:
It would, but this was just me trying to see why `wasm-opt` wasn't being picked up. As I mentioned here (https://github.com/Homebrew/homebrew-core/pull/80482#issuecomment-873416027) I found out later that the function documentation was outdated. There's a patch upstream to fix this, I'm trying to see if that along with some changes in the test code could get this to work.

We would probably have to use `--mode=no-install` in the test because our `binaryen` is newer than the one they require. It should be backwards-compatible though.